packages-proprietary/react-icons/src/VotingBallot.tsx

```diff
@@ -1,11 +1,13 @@
 import type { SVGProps } from 'react'
 
 const SvgVotingBallot = (props: SVGProps<SVGSVGElement>) => (
-  <svg aria-hidden="true" focusable="false" viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg" {...props}>
+  <svg aria-hidden="true" focusable="false" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" {...props}>
```

Comment:
This made me find a few more icons that have a 32×32 viewbox: Airplane, Broom, HandWithEuroCoin (deprecated), MarketStall, PersonPraying. Can we resize these to 24×24 as well?

Reply:
Those are the legacy icons, I did not change those to nog break old implementations. This would require me to add them to Figma and export them again.

Reply:
Airplane, Broom, MarketStall aand PersonPraying are awaiting updates, they will receive an updated viewbox then.

Reply:
This is what you'll when you just change the viewbox:
<img width="198" height="197" alt="image" src="https://github.com/user-attachments/assets/bcc24979-c397-462e-8818-3d4d381e718e" />